python-checks/src/test/resources/checks/argumentType.py

```diff
@@ -59,25 +59,27 @@ def stdlib_functions():
   A = UnexpectedClass()
   acos(A) # Noncompliant
   B = datetime.tzinfo()
-  B.tzname(42) # Noncompliant {{Change this argument; Function "tzname" expects a different type}}
-#          ^^
+  B.tzname(42) # FN Noncompliant {{Change this argument; Function "tzname" expects a different type}}
+##          ^^
   select.select([],[],[], 0)
   time.sleep(1) # OK
   x = time.gmtime(int(time.time()))
   x = time.gmtime(secs = int(time.time()))
   time.sleep(True) # OK, converted to 1
   time.sleep(1j) # FN, considered duck type compatible
   genericpath.isfile("some/path")
-  genericpath.isfile(42) # Noncompliant
+  genericpath.isfile(42) # FN Noncompliant
   my_list = [1,2,3]
   _heapq.heapify(42) # Noncompliant {{Change this argument; Function "heapify" expects a different type}}
 #                ^^
   _heapq.heapify(my_list)
   imap4 = imaplib.IMAP4()
-  imap4.setannotation(42) # FN, we do not handle variadic parameters
+  # imap4 is not part of our protobufs anymore but it should be an FN, we do not handle variadic parameters
+  imap4.setannotation(42) # Noncompliant
```

Comment:
I'm not sure how we resolve this then? We resolve a call to `imaplib.IMAP4.setannotation`, but we do not see that the argument is variadic for some reason.

Reply:
This is solved with 6c16fd10dd402bc04e629a6aa3fdfda877f2fee1